test-cases/cdc/cdc-15m-replication-postimage.yaml

```diff
@@ -1,5 +1,5 @@
 test_duration: 120
-
+region_name: "us-east-1"
```

Comment:
why does the `region_name` need to be hard coded in this file?

Reply:
Not needed. I investigated why unit test fail, and forgot to remove. fixed